### PR TITLE
Add biaodian - the chinese punctuation marks extension

### PR DESCRIPTION
**Pull request by gasolin**
## Description

Add the Chinese punctuation marks (中文標點符號, `biaodian` is the spell of `標點` in chinese)
FYR https://www.ltl-shanghai.com/chinese-punctuation/

some common punctuation marks (ex: ，。 )are able to type via keyboard with input method (IME), but some (ex:、「」) are not that easy.

With this extension you can list or search via english punctuation marks (ex: . , [ ]) to find the chinese punctuation marks (。，『』)

## Type of change

- New extension (based on gitmoji)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/biaodian/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "name": "biaodian",
+  "title": "Search Chinese Punctuation Marks",
+  "description": "Search Chinese Punctuation Marks",
+  "icon": "command-icon.png",
+  "author": "gasolin",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "biaodian",
+      "title": "Biaodian (Chinese Punctuation Marks)",
```
Comment (pernielsentikaer) [inline extensions/biaodian/package.json:12]:
```suggestion
      "title": "Biaodian",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @gasolin 👋 

I only have a single suggestion before it should be fine to ship 🚢 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by gasolin** [comment]
@pernielsentikaer updated, thanks

File: extensions/biaodian/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "name": "biaodian",
+  "title": "Search Chinese Punctuation Marks",
+  "description": "Search Chinese Punctuation Marks",
+  "icon": "command-icon.png",
+  "author": "gasolin",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "biaodian",
+      "title": "Biaodian",
```
Comment (pernielsentikaer) [inline extensions/biaodian/package.json:12]:
```suggestion
      "title": "Biaodian",
      "subtitle": "Chinese Punctuation Marks",
```
Sorry I forgot to add the subtitle, could you add?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @gasolin 👋 

Forgot that there was no subtitle, could you add suggested 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by gasolin** [comment]
@pernielsentikaer added, please kindly review again

**Review by pernielsentikaer — APPROVED** [review]
hi @gasolin 👋 

Much better 💯  I'm going to accept it and then will my colleague do a final review before it's finally merged 💎

**Review by mathieudutour — APPROVED** [review]
Looking good, well done :+1:

**Pull request merged**